partial-update/pom.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright (c) Microsoft Corporation. All rights reserved. -->
+<!-- Licensed under the MIT License. -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.azure.tools</groupId>
+        <artifactId>azure-autorest-parent</artifactId>
+        <version>1.0.0-beta.1</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+    <groupId>com.azure.tools</groupId>
+    <artifactId>azure-autorest-partial-update</artifactId>
+    <version>1.0.0-beta.1</version>
+
+    <repositories>
+        <repository>
+            <id>azuresdkmaven</id>
+            <url>https://azsdkartifacts.blob.core.windows.net/maven</url>
+        </repository>
+    </repositories>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-simple</artifactId>
+            <version>1.7.36</version>
+        </dependency>
```

Comment:
As a lib, you don't need this. Or only in test?

Reply:
It;s easier for testing, I saw cadl-extension has this dependency, so I add it, can be deleted.

Reply:
as said, if used for test, put it to test scope.

cadl-extention is the app, that being called by emitter.

this is a module that not directly called as jar, but used in other module or app. hence what is the appropriate logging (std, file, or else) is the decision of the final app.

Reply:
sure, I will  update, thanks